docs/process_to_manipulate_data.md

```diff
@@ -0,0 +1,44 @@
+### Process to manipulate data in production
+
+To manipulate data in production, the default workflow is to create a `sqitch` change, associated with `pgTap` tests that are run in our CI pipeline and tested in our `dev` and `test` environments as the application is deployed.
+
+In rare cases, the team needs to manipulate data directly in the database, without going through our usual guardrails. Examples of this are mostly around manipulation of private data. This repository is open-source, therefore we should not commit any private information.
+
+This should be considered a last resort. Updating data in production could lead to cascading failure if done without the proper guardrails, which our CI are providing.
+#### Important steps before you begin:
+
+- Get request/approval from Product Owner
+- Back up database before manipulating data
+- Pair program with another developer
+
+#### Manipulating the data
+
```

Comment:
do we want a command line to copy-paste to find the leader pod?
Something like: 
```
oc project ff61fb-dev
oc -n ff61fb-dev get pods -l postgres-operator.crunchydata.com/cluster=ccbc,postgres-operator.crunchydata.com/role=master
oc -n ff61fb-dev exec -it <db-leader-pod-name> -- psql -d ccbc
```
And a note:

> The `oc project` command and `-n` option in subsequent commands are redundant. They act as an additional guardrail to prevent accidentally running commands in the wrong namespace

Reply:
I'm for a copy-paste to find the leader pod

Reply:
Alright I added a section with this